marketplace/src/main/assemble/assembly.xml

```diff
@@ -84,28 +84,66 @@
       <patternset refid="easyshare.versus.jsf" />
     </unzip>
 
+    <!-- Separate JSF packages -->
+    <mkdir dir="${outdir}/marketplace/install/bundles-jsf-ui" />
+    <move todir="${outdir}/marketplace/install/bundles-jsf-ui">
+      <fileset dir="${outdir}/marketplace/install/bundles">
+        <include name="nuxeo-easyshare-jsf-*.jar" />
+      </fileset>
+    </move>
+
+    <!-- Separate Web UI packages -->
+    <mkdir dir="${outdir}/marketplace/install/bundles-web-ui" />
+    <move todir="${outdir}/marketplace/install/bundles-web-ui">
+      <fileset dir="${outdir}/marketplace/install/bundles">
+        <include name="nuxeo-easyshare-web-ui-*.jar" />
+      </fileset>
+    </move>
+
     <!-- Generate install.xml file -->
-    <var name="install.content" value="&lt;install&gt;" />
-    <var name="install.content"
-         value="${install.content}${line.separator}
-    &lt;update file=&quot;${package.root}/install/bundles&quot; todir=&quot;${env.bundles}&quot; /&gt;" />
+    <var name="install.bundles" value=""/>
+    <var name="install.bundles"
+         value="${line.separator}
+  &lt;update file=&quot;${package.root}/install/bundles&quot; todir=&quot;${env.bundles}&quot; /&gt;"/>
+    <var name="install.bundles"
+         value="${install.bundles}${line.separator}
+  &lt;update file=&quot;${package.root}/install/bundles-jsf-ui&quot; todir=&quot;${env.bundles}&quot; if=&quot;Packages.contains('nuxeo-jsf-ui')&quot; /&gt;"/>
+    <var name="install.bundles"
+         value="${install.bundles}${line.separator}
+  &lt;update file=&quot;${package.root}/install/bundles-web-ui&quot; todir=&quot;${env.bundles}&quot; if=&quot;Packages.contains('nuxeo-web-ui')&quot; /&gt;" />
+    
+    <var name="install.libs" value="" />
     <if>
       <available file="${outdir}/marketplace/install/lib" />
       <then>
-        <var name="install.content"
-             value="${install.content}${line.separator}
-    &lt;update file=&quot;${package.root}/install/lib&quot; todir=&quot;${env.lib}&quot; /&gt;" />
+        <var name="install.libs"
+             value="${line.separator}
+  &lt;update file=&quot;${package.root}/install/lib&quot; todir=&quot;${env.lib}&quot; /&gt;" />
+      </then>
+    </if>
+    <if>
+      <available file="${outdir}/marketplace/install/lib-jsf-ui" />
+      <then>
+        <var name="install.libs"
+             value="${install.libs}${line.separator}
+  &lt;update file=&quot;${package.root}/install/lib-jsf-ui&quot; todir=&quot;${env.lib}&quot; if=&quot;Packages.contains('nuxeo-jsf-ui')&quot; /&gt;" />
```

Comment:
Missing install.libs for web ui. Take a look at what's being done on [DAM](https://github.com/nuxeo/marketplace-dam/blob/master/marketplace/src/main/assemble/assembly.xml#L158-L165).